src/StyledComponents.js

```diff
@@ -0,0 +1,75 @@
+import { styled } from "@mui/material/styles";
+import Paper from "@mui/material/Paper";
+import customTheme from "./theme";
+import { Box, Button, Divider, Link } from "@mui/material";
+
+const ContactItem = styled(Paper)(({ theme }) => ({
+  padding: theme.spacing(1),
+  textAlign: "center",
+}));
+export default ContactItem;
+
+const HeroTitle = styled(Box)(({ theme }) => ({
+  padding: theme.spacing(1),
+  variant: "h3",
+  background: customTheme.palette.secondary.main,
+  fontSize: 60,
+  border: "none",
+  fontWeight: 600,
+  position: "relative",
+  textAlign: "start",
+  mt: "12%",
+  color: customTheme.palette.primary.main,
+  mb: "5%",
+}));
+export { HeroTitle };
+
+const MenuItemLink = styled(Link)(({ theme }) => ({
+  color: "#fff",
```

Comment:
can we use theme colors, theme.palette.common.white

Reply:
done